### ISSUE TITLE
Latest versions does not filter by --tags

### ISSUE THREAD

**Issue by xescugc** [cycloidio/terracognita#322]
**General information:**

* Operating System: 
* Terracognita version / tag: 0.8.1
* Did you build Terracognita from sources or did you use the Docker image: 

**Describe the bug:**

The tags passed via flags are not taken into consideration when importing

**Log message**

Here you can paste the log message or paste the link to console logs. If the log message is too big, you can use a tool like https://pastebin.com/.

With `0.7.6` I can do this:
```
~ $> terracognita aws --aws-profile cycloid --aws-default-region eu-west-1 --tags test:key --hcl out.tf
Starting Terracognita with version v0.7.6
Importing with filters:
Tags:    [{test key}],
Include: [],
Exclude: [],
Targets: [],
Importing aws_alb [1/2]^C
```

But for the latest one I cannot
```
~ $> terracognita aws --aws-profile cycloid --aws-default-region eu-west-1 --tags test:key --hcl out.tf
Starting Terracognita with version v0.8.1
Importing with filters:
Tags:    [],
Include: [],
Exclude: [],
Targets: [],
Importing aws_instance [1/38]^C
```

**Additional context**

**Comment by jmurillo9** [comment]
Can we please get a working example of what the proper syntax for this is? I cannot seem to get it to work.

```bash
[jon@laptop]$ terracognita version
The current version is: v0.8.1
```

I've tried using `-t "key:value"` , `-t 'key:value`, `-t key:KEY,value:VALUE` and `-t "{key:KEY,value:VALUE}"`

**Comment by Boggimedes** [comment]
@jmurillo9 @xescugc 
Same issue.  Tags are not working with -t or --tags no matter what format I use.

**Comment by yogeshnegi-srijan** [comment]
Yes getting the same issue for me

**Comment by xescugc** [comment]
Sorry all for the delay, it works with `--tags key:value,key2:value2` or `--tags key:value --tags key2:value2`.

The main reason why it was still failing after this was merged is because we did not do a release and so the 0.8.1 still had it but it was merged with the 0.8.2, sorry for that :pray:
